Kinvey/Kinvey/HttpRequest.swift

```diff
@@ -343,9 +343,7 @@ internal class HttpRequest: TaskProgressRequest, Request {
         let url = endpoint.url
         request = URLRequest(url: url)
         request.httpMethod = httpMethod.stringValue
-        if let timeout = options?.timeout {
-            request.timeoutInterval = timeout
-        }
+        request.timeoutInterval = options?.timeout ?? client.options?.timeout ?? client.timeoutInterval
```

Comment:
yes, indeed is begin used... the ?? operator try to use the value in the left side first, if there's no value then it uses the value in the right side